src/makeCalculator.js

```diff
@@ -37,7 +37,38 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    add(a, ac) {
```

Comment:
a, ac??? I don't understand what means these letters, maybe try to use some words.